src/insns/cgetlen_32bit.adoc

```diff
@@ -19,7 +19,7 @@ Calculate the length of `cs1` 's bounds and write the result in `rd`. The length
 is defined as the difference between the decoded bounds' top and base addresses
 i.e. `top - base`. It is not required that the input capability `cs1`  has its
 tag set to 1. <<CGETLEN>> outputs 0 if `cs1` 's bounds are malformed (see
-xref:section_cap_malformed[xrefstyle=short]).
+xref:section_cap_malformed[xrefstyle=short]), or if the length of `cs` is 2^XLENMAX^.
```

Comment:
Unless it's been changed in this spec, 2^XLENMAX actually gets output as all-1s so the mapping is injective

Reply:
Interesting.  Did that prove more useful in practice than "NULL reads as zero bytes"?

Reply:
@jrtc27 : The -1s idea is kept in this specification. This is written in the CGETLEN description from Chapter 3; it was simply forgotten here.